common/src/test/java/com/vexsoftware/votifier/net/protocol/TestVotifierPlugin.java

```diff
@@ -75,6 +80,16 @@ public String getVersion() {
         return "2.3";
```

Comment:
See comment above about protocol versions

Reply:
I'll just change this to "2".